### ISSUE TITLE
"Ghost" UI elements staying visible while editing relations

### ISSUE THREAD

**Issue by mboeringa** [openstreetmap/iD#4608]
While editing relations, I noticed in the left side bar for the tagging, that regularily, a kind of "ghost" UI elements stayed visible. This is confusing. These boxes are related to a previous edit of another way, but initially keep visible when I switch / select another way in the editor's viewing window.

It seems some refresh is necessary?

![afbeelding](https://user-images.githubusercontent.com/7635726/33795148-c1af76bc-dcdb-11e7-9fec-49366259a31d.png)

**Comment by bhousel** [comment]
Thanks for reporting @mboeringa ... Those are leftover dropdown menus and should definitely go away once the dropdown is gone.  What browser/os are you using?

**Comment by mboeringa** [comment]
This was with the latest Firefox 57.0.2 on Windows 10.

**Comment by mboeringa** [comment]
@bhousel:

Late response, but I am now suspecting the particular issue may have been due to some background processing going on. I had started some batch process before this issue occurred, and it turned out to have some bigger impact on the load of CPU and GPU than I initially expected. I think this is the main cause of what I witnessed, although it still slightly puzzles me that I could switch to another object, and see the main dropdown update, while these "ghost" images kept visible.

Anyway, unless you were able to reproduce it, I think I am going to close this issue for now.

Marco
